docs/griptape-framework/structures/observability.md

```diff
@@ -0,0 +1,56 @@
+## Overview
+
+The [Observability](../../reference/griptape/observability/observability.md) context manager sends telemetry (metrics and traces) for all functions and methods annotated with the `@observable` decorator to a destination of your choice. This is useful for monitoring and debugging your application.
+
+Observability is completely optional. To opt in, provide an `ObservabilityDriver` to the `Observability` context manager, for example:
```

Comment:
Link `ObservabilityDriver` to `drivers/observability-drivers.md`

Reply:
Removed reference to ObservabilityDriver, so no longer applicable.